connector/src/main/java/org/geysermc/connector/utils/ResourcePack.java

```diff
@@ -0,0 +1,73 @@
+package org.geysermc.connector.utils;
+
+import org.geysermc.connector.GeyserConnector;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.zip.ZipFile;
+
+public class ResourcePack {
```

Comment:
I would add a small Javadoc comment here explaining the purpose of this class, e.g.
```java
/**
* Represents a resource pack in the `packs` folder.
*/
```